### ISSUE TITLE
qregularexpression.h not found on Fedora

### ISSUE THREAD

**Issue by Evidlo** [ros-visualization/python_qt_binding#42]
Compilation fails with the message 

```
/usr/share/sip/PyQt5/QtCore/qregularexpression.sip:26:32: fatal error: qregularexpression.h: No such file or directory
compilation terminated.
```

`src/python_qt_binding/cmake/sip_configure.py` is hardcoded to use the `qmake` binary without doing any version checking.  On Fedora, it's actually called `qmake-qt5` and `qmake` doesn't exist.

Another user using Slackware [ran into the same issue](http://barelywalking.com/?p=13).

Maybe the script should look for `qmake-qt5` first, then try `qmake` (with version checking).

**Comment by dirk-thomas** [comment]
> `src/python_qt_binding/cmake/sip_configure.py` is hardcoded to use the `qmake` binary without doing any version checking. On Fedora, it's actually called `qmake-qt5` and `qmake` doesn't exist.

Shouldn't it actually fail with something like "qmake not found" rather than a header not being found?

> Maybe the script should look for `qmake-qt5` first, then try `qmake` (with version checking).

Please consider to provide a pull request for this.

**Comment by dhood** [comment]
`qmake` is typically just a symlink to `qtchooser` on ubuntu (apparently this is different with fedora). Do you have qtchooser installed? You might just need to configure it to pass through to the `qmake-qt5` executable, instead of modifying the code

**Comment by Evidlo** [comment]
@dirk-thomas You're right.  The error is different because I had manually symlinked qmake to qtmake-qt4.  The issue remains however.

@dhood Installing the `qtchooser` package fixed the issue.  Is this an issue with rosdep?

**Comment by riebl** [comment]
I have stumbled across the same issue, however, there is no qtchooser on openSUSE Leap 42.3 (at least I am not aware of it). #43 fixes this build issue on my system by calling qmake-qt5 if available and falling back to qmake. In my opinion this should not break builds on Ubuntu.

**Comment by dirk-thomas** [comment]
Addressed by #43.
